src/bin/vip-logs.js

```diff
@@ -202,7 +207,7 @@ command( {
 	module: 'logs',
 } )
 	.option( 'type', 'The type of logs to be returned: "app" or "batch"', 'app' )
-	.option( 'limit', 'The maximum number of log lines', 500 )
+	.option( 'limit', `The maximum number of log lines (defaults to ${ LIMIT_DEFAULT })` )
```

Comment:
Minor: I wonder if we should add a comment to explain why we're setting the default in this way rather than via the arg?

Reply:
Yes, it would definitely help future contributors. Addressed in [2f7ad73](https://github.com/Automattic/vip-cli/pull/2093/commits/2f7ad739115684de10183063efed7ab9af891044)